src/history/types.ts

```diff
@@ -23,5 +23,5 @@ export interface FilterButtonProps {
   readonly type: string
   readonly value: string
   readonly selected: boolean
-  readonly onClick: (type: string, value: string) => void
+  readonly onClick: ClickFilterHistory
```

Comment:
`ClickFilterHistory` implies that it's an action. IMO `FilterHistoryClickHandler` would make more sense given it's a type.